ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -260,10 +262,20 @@ struct FrameworkBuilder {
       """)
 
       // Use the Xcode-generated path to return the path to the compiled library.
-      let buildName = frameworkBuildName(framework)
-      let fileName = LaunchArgs.shared.dynamic ? "\(buildName).framework" : "lib\(buildName).a"
-      let libPath = buildDir.appendingPathComponents(["Release-\(platformFolder)", framework,
-                                                      fileName])
+      let frameworkPath = buildDir.appendingPathComponents(["Release-\(platformFolder)", framework])
+      var actualFramework: String
+      do {
+        let files = try FileManager.default.contentsOfDirectory(at: frameworkPath,
+                                                                includingPropertiesForKeys: nil).compactMap { $0.absoluteString }
+        let frameworkDir = files.filter { $0.contains(".framework") }
+        actualFramework = URL(fileURLWithPath: frameworkDir[0]).lastPathComponent
+      } catch {
+        fatalError("Error while enumerating files \(frameworkPath): \(error.localizedDescription)")
+      }
+      var libPath = frameworkPath.appendingPathComponent(actualFramework)
+      if !LaunchArgs.shared.dynamic {
+        libPath = libPath.appendingPathComponent(actualFramework.replacingOccurrences(of: ".framework", with: ""))
```

Comment:
Some brief explanation why this (and above) path transformation works will help us in the future. Maybe a add a dir structure in the comments or smth.